### ISSUE TITLE
Enable bitcode for iOS framework not working

### ISSUE THREAD

**Issue by emysa341** [mono/Embeddinator-4000#698]
### Steps to Reproduce

1. While embedinating a C# source with following arguments: 
**mono** objcgen.exe **--gen**=obj-C **--platform=ios** -c -d -v --nativeexception **--bitcode**=true **--target**=framework **-o**=outputpath  MyInputDllFile.dll

2. Embedination is successful, upon using that framework in a project, works totally fine if I disable the bitcode. If bitcode is enable in the project it gives me an error that the framework (generated by E4k) does has bitcode enabled. How we can embeddinate with bitcode enabled ? I already set --bitcode=true but it does not work. Also, the framework produced after embeddination has file size of around 300MB. How I can reduce the framework size ?

### Expected Behavior
E4k should embeddinate in a way that iOS.framework has bitcode enabled.
### Actual Behavior
E4k embeddinates but output framework has no bitcode feature.
### Environment
MacOS, VS Mac

```
```

### Build Logs
ld: ‘TicketingServiceIos.framework/TicketingServiceIos’ does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. file ‘TicketingServiceIos.framework/TicketingServiceIos’ for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by mandel-macaque** [comment]
@emysa341 It looks like that have been fixed as commented in https://github.com/mono/Embeddinator-4000/issues/513 can you double check that you are using the version that contains the mentioned PR. Thanks!

**Comment by rolfbjarne** [comment]
Xamarin.iOS currently only supports bitcode for watchOS and tvOS, not iOS, so this won't work for now (until Xamarin.iOS adds support for bitcode on iOS, which is a known issue but currently not a high priority, so it may take a while).

**Comment by ColeXm** [comment]
Does bitcode  support on iOS now ?

**Comment by hammadahmed1985** [comment]
Any updates on this? Are there any plans to support bitcode for Xamarin.iOS?

**Comment by rolfbjarne** [comment]
@hammadahmed1985 no, no plans to support bitcode on iOS for now, unless Apple makes it mandatory.
